### PR TITLE
Prepare release 0.2.0

### PR DESCRIPTION
**Pull request by agentk**
I'm marking this as 0.2.0 because of the subset support enhancement.

Somebody want to check the release over and merge if ok? // @sgen @julianxhokaxhiu @perrin4869

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Path to the font files relative to the css generated.
 
 Default = ''
 
-### subset
+### subset:
 
 A string or array with the characters desired to be included inside the generated fonts
```
Comment (perrin4869) [inline README.md]: Add backticks here?

File: README.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ Path to the font files relative to the css generated.
 
 Default = ''
```
Comment (perrin4869) [inline README.md]: And backtick here?

File: README.md [hunk 3]
```diff
@@ -130,3 +130,21 @@ Note: If present, the json config file must be valid json.
             });
         }
     };
+
+# Changelog
```
Comment (perrin4869) [inline README.md]: Maybe separate Changelog into its own file, `CHANGELOG.md`?

**Comment by agentk** [comment]
Good call. I forgot to markup the parameters too.

**Comment by perrin4869** [comment]
LGTM :)

**Comment by agentk** [comment]
Cheers, published. Thanks for your work @perrin4869. Great to see.

**Comment by julianxhokaxhiu** [comment]
Awesome! I didn't saw this PR in time, but definitely happy of it :)

**Pull request merged**